themes/SuiteP/css/suitep-base/main.scss

```diff
@@ -25,6 +25,7 @@ h3 {
 // override bootstrap
 a {
   color: $link-color;
+  overflow-wrap: anywhere;
```

Comment:
Hi @serhiisamko091184, this is a very generic place in the code. Its hard to measure the impacts on rules as generic as this one. It would be best to have a rule just for fields, or maybe even just for url fields.

Reply:
Thanks for feedback, @clemente-raposo 
I did some changes: 
![image](https://github.com/salesagility/SuiteCRM/assets/116086008/03f45e09-1280-4946-a6e6-9b9cbee3a0a0)

![image](https://github.com/salesagility/SuiteCRM/assets/116086008/f558a36c-5a5e-4910-848a-3de1caec5843)

Thanks in advance for review! 

Regards,
Serhii